apps/registry/src/index.ts

```diff
@@ -22,6 +19,20 @@ async function runHealthCheck(ns: DurableObjectNamespace, version: string) {
 
 export default {
   async fetch(request: Request, env: RegistryDOEnv): Promise<Response> {
+    const url = new URL(request.url);
+
+    if (url.searchParams.get('blockNumber') !== '') {
+      const network = url.searchParams.get('network') as Network;
+      const blockNumber = parseInt(url.searchParams.get('blockNumber'));
+      const server = new Servers.OracleRegistryServer();
+      const oracles = await server.queryAllOracles(
```

Comment:
Instead of calling these two private methods directly can you change this to use a generic `refreshAtBlock` method on `server-registry.js` directly? This will be more generic in the future

```
  // INSIDE base-registry.ts
  protected abstract _refresh(
    network: Network,
    intervalNum: number,
    blockNumber?: number
  ): Promise<CacheSchema<T>>;  

  // INSIDE server-registry.ts
  public async refreshAtBlock(network: Network, blockNumber: number, intervalNum = 0) {
     return  JSON.stringify(await this._refresh(network, intervalNum, blockNumber)));
   }
```